nbs/sidebar.yml

```diff
@@ -0,0 +1,35 @@
+website:
```

Comment:
You should remove this file - we're not using it any more.

Reply:
ok removed it!